src/ansys/mechanical/core/embedding/app.py

```diff
@@ -218,6 +234,10 @@ def _dispose(self):
         self._app.Dispose()
         self._disposed = True
 
+    def is_initialized(self):
```

Comment:
this should be a free function. otherwise you need an instance of the app to check if the app is initialized